fluentd/run.sh

```diff
@@ -18,8 +18,6 @@ done
 
 echo "============================="
 echo "Fluentd logs have been redirected to: $LOGGING_FILE_PATH"
-echo "If you want to print out the logs, use command:"
-echo "oc exec <pod_name> $HOME/utils/logs"
```

Comment:
I don't think we want to get rid of this - the message should read
`If you want to print out the logs, use command:`
`oc exec <pod_name> -- logs`

Reply:
Ok.  Done.
So, for bz1643340, there's no more to fix!?

Reply:
There were a couple of other issues reported in that bz

> rsh into a fluentd pod, there is no fluentd.log file in /var/log/fluentd directory

Can you confirm that this is working?

> No output shows after run "oc exec $fluentd-pod -- logs".

Can you confirm that this is working?

One more thing you might do if you want to learn some bash tricks - make the output of the run.sh script go to the /var/log/fluentd/fluentd.log file.  That is - the only thing going to stdout will be the message about the logging file path.  Then, close stdout/stderr, and redirect stdout/stderr to /var/log/fluentd/fluentd.log for the rest of the process.

```bash
exec >> $LOGGING_FILE_PATH 2>&1
```
that will make all stdout/stderr from all commands go to `$LOGGING_FILE_PATH`

Reply:
>There were a couple of other issues reported in that bz
` rsh into a fluentd pod, there is no fluentd.log file in /var/log/fluentd directory`
> Can you confirm that this is working?

Yes, there are fluentd.log file and its rotated files in /var/log/fluentd directory in v3.11 openshift:
```
# oc exec logging-fluentd-dflrc -- ls -l /var/log/fluentd
total 9904
-rw-r--r--. 1 root root  887959 Nov 14 18:53 fluentd.log
-rw-r--r--. 1 root root 1024387 Nov 14 18:52 fluentd.log.0
-rw-r--r--. 1 root root 1025148 Nov 14 18:52 fluentd.log.1
-rw-r--r--. 1 root root 1024609 Nov 14 18:51 fluentd.log.2
```
`No output shows after run "oc exec $fluentd-pod -- logs".`
> Can you confirm that this is working?

Yes, it is working as expected.

> One more thing you might do if you want to learn some bash tricks - make the output of the run.sh script go to the /var/log/fluentd/fluentd.log file. That is - the only thing going to stdout will be the message about the logging file path. Then, close stdout/stderr, and redirect stdout/stderr to /var/log/fluentd/fluentd.log for the rest of the process.
`exec >> $LOGGING_FILE_PATH 2>&1`
> that will make all stdout/stderr from all commands go to $LOGGING_FILE_PATH

All right.  Let me test it...